make_release/Readme.md

```diff
@@ -1,8 +1,22 @@
 # The release process of Nushell
 ## 0. Check dependencies
-- [ ] [`reedline`] is updated and released ([example][reedline bump example])
-- [ ] pin [`reedline`] in [`nushell`] ([example][reedline pin example])
-- [ ] [`nu-ansi-term`] stays the same
+> **Note**  
+> the following procedure is the same for `nu-ansi-term` and `reedline` and needs to be repeated
+
+> **Warning**  
+> release `nu-ansi-term` **before** `reedline` and `reedline` **before** Nushell
+
+> **Note**  
+> `nu-ansi-term` is typically released only when there are changes to publish.
+> `reedline` is typically released on the same schedule as Nushell.
+
+- [ ] bump the version (example with [`reedline`][reedline bump example] and [`nu-ansi-term`][nu-ansi-term bump example])
+- [ ] get the latest revision with `git pull origin main`
+- [ ] publish the crate with `cargo publish` (*need to be a member of the publishing team*)
+- [ ] tag the project with `git tag ...` (`0.xx.0` for `reedline` and `v0.xx.0` for `nu-ansi-term`)
```

Comment:
The tags for reedline are also `v` prefixed. Only the github release pages skip that. `nushell/nushell` is the outlier here.

Reply:
i've been fooled by the *GitHub* releases :scream: 

should be better with 300a50a :wink:

Reply:
Thanks!